lib/docassemble/get_base_interview_url.js

```diff
@@ -11,9 +11,18 @@ let get_base_interview_url = async function () {
   *    when the user is not logged in.) We can't use `reset=2` because it caused an
   *    'Unable to locate interview session' error. */
   let server_url = session_vars.get_da_server_url();
-  let dev_id = await docassemble_api_interface.get_dev_id();
-  let project_name = session_vars.get_project_name();
-  return `${ server_url }/interview?new_session=1&i=docassemble.playground${ dev_id }${ project_name }%3A`;
+  console.log(session_vars.get_install_type());
+  if (session_vars.get_install_type() === "server") {
+    let x = `${ server_url }/interview?new_session=1&i=${session_vars.get_package_name().replace("-", ".")}%3A`;
+    console.log(`base_interview_url: ${x}`)
+    return x;
+  } else {
+    let dev_id = await docassemble_api_interface.get_dev_id();
+    let project_name = session_vars.get_project_name();
+    let x = `${ server_url }/interview?new_session=1&i=docassemble.playground${ dev_id }${ project_name }%3A`;
```

Comment:
Discussed on deep dive, but another refactor I would consider if there should be a different way of passing along this URL. Not sure the best way though.

Reply:
Thoughts: We're always going to be the ones deriving the URL. I think we were just describing separating the concerns. We could make a script that just determines the url of an interview installed on the server and a script that determines the url of a playground interview. Each action could use the appropriate one and output the result to the action, which would then use it in the command line arguments.

Reply:
We'll have to have a real-time discussion to refresh my memory on this. Right now my confused brain is wondering why we need to create the whole url ourselves. For example, couldn't we send the base url of the interview through the command line. Maybe that's still us - ALKilnInThePlayground is us - but there's still a distinction, right? Allow that repo to send the base url through the command line? Is that the flexibility we're looking for? So that we, as developers, can pass in the base url of the interview through the command line?

Maybe we should write some user stories to help clarify the exact paths we're thinking of.

Reply:
Conclusion: There were only 2 url types - server install and playground install, so we'll handle it by using an npm script to install a runtime config var we can use as a flag.